torch/autograd/profiler.py

```diff
@@ -116,55 +116,56 @@ def export_chrome_trace(self, path):
+                        '"args": {}}, ' % (evt.name, evt.cpu_interval.start,
+                                           evt.cpu_interval.elapsed_us(), evt.thread))
                 for k in evt.kernels:
                     # 's' and 'f' draw Flow arrows from
                     # the CPU launch to the GPU kernel
-                    chrome_events.append(dict(
-                        name=evt.name,
-                        ph='s',
-                        ts=evt.cpu_interval.start,
-                        tid=evt.thread,
-                        pid='CPU functions',
-                        id=next_id,
-                        cat='cpu_to_cuda',
-                        args={},
-                    ))
-                    chrome_events.append(dict(
-                        name=k.name,
-                        ph='f',
-                        ts=k.interval.start,
-                        tid=k.device,
-                        pid='CUDA functions',
-                        id=next_id,
-                        cat='cpu_to_cuda',
-                        args={},
-                    ))
-                    chrome_events.append(dict(
-                        name=k.name,
-                        ph='X',
-                        ts=k.interval.start,
-                        dur=k.interval.elapsed_us(),
-                        tid=k.device,
-                        pid='CUDA functions',
-                        args={},
-                    ))
+                    f.write('{"name": "%s", '
+                            '"ph": "s", '
+                            '"ts": %s, '
+                            '"tid": %s, '
+                            '"pid": "CPU functions", '
+                            '"id": %s, '
+                            '"cat": "cpu_to_cuda", '
+                            '"args": {}}, ' % (evt.name, evt.cpu_interval.start,
+                                               evt.thread, next_id))
+                    f.write('{"name": "%s", '
+                            '"ph": "f", '
+                            '"ts": %s, '
+                            'tid": %s, '
+                            '"pid": "CUDA functions", '
+                            '"id": %s, '
+                            '"cat": "cpu_to_cuda", '
+                            '"args": {}}, ' % (k.name, k.interval.start, k.device, next_id))
+                    f.write('{"name": "%s", '
+                            '"ph": "X", '
+                            '"ts": %s, '
+                            '"dur": %s, '
+                            '"tid": %s, '
+                            '"pid": "CUDA functions", '
+                            '"args": {}}, ' % (k.name, k.interval.start,
+                                               k.interval.elapsed_us(), k.device))
                     next_id += 1
 
-            json.dump(chrome_events, f)
+            # remove trailing whitespace and comma
+            f.seek(f.tell()-2, os.SEEK_SET)
```

Comment:
@v0dro the linter is complaining about this line, should have spaces around the minus operator.

Reply:
Other CI failures are all unrelated, so this is good to go with that one trivial fix.